gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -890,6 +890,7 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
       int i;
       gchar **strv = NULL;
 
+      /** TODO: update layout here */
```

Comment:
With tensor_transform, we may accept (as syntatic suger) ops as: "data-layout-change-to-whs" (this is just a verbose example. the real syntax should be simpler)